index.html

```diff
@@ -0,0 +1,275 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tasnimul Hasan Portfolio</title>
+    <link href="styles.css" rel="stylesheet" type="text/css">
+    <style>
+    </style>
+</head>
+<body>
+
+    <header>
+        <div class="nav-menu-content">
+
+            <div class="logo"><a class=mylogo> My Logo </a></div>
+
+            <nav>
+                <ul class="nav-menu">
+                    <li class="nav-items"><a href="#">Portfolio</a></li>
+                    <li class="nav-items"><a href="#">About</a></li>
+                    <li class="nav-items"><a href="#">Contact</a></li>
+                  </ul>
+
+
+                    <div class="hamburger">
+
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                        <span class="bar"></span>
+                    </div>
+            </nav>
+        </div>
+
+    </header>
+    <section class="homepage">
+        <div class="introduction">
+            <div class="intro"><h1>I'm Tasnimul <br>Glad to meet you!</h1></div>
+            <div class="me">
+                <p>
+                    I am a software developer. I have done some interesting projects using
+                    react js, node js and ruby on rails. I am learning new things and
+                    implementing them instantly. I can help you build a project, feature
+                    or website look through some of work and experience if you like what
+                    you see and have a project you need coded, don't hesitate to contact
+                    me.
+                </p>
+            </div>
+            <div class="connect">
+                <h3><a> Let's connect </a> </h3>
+            </div>
+            <div class="social">
+                <ul>
+                    <li class="icon"><a href=""><img src="Image/twitter.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/github.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/medium.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/angellist.png" alt="" /></a></li>
+                    <li class="icon"><a href=""><img src="Image/Linkedinicon.png" alt="" /></a></li>
```

Comment:
I think the issue resides in [here](https://stackoverflow.com/questions/21306512/difference-between-relative-path-and-absolute-path-in-javascript). Your path is absolute so that's why your images are not loading in the tools we use for code reviewing. I checked your project locally and the image are loading up.